Probesys/bileto
# imp: Display the attached documents

## Related issue(s)



https://github.com/Probesys/bileto/issues/149

## Changes



- Add a `MessageDocument::getType` method and return it when uploading a file in order to display a coherent icon
- Add "file" icons 
- Display the attached images added with TinyMCE under the editor
- Set the `MessageDocument::message` attribute when creating a ticket or when posting an answer
- Display the attached documents in the messages (timeline)

## How to test manually



- create a ticket/post an answer and add an image to your message → check it appears below the editor
- once posted → check it appears at the bottom of the message

## Checklist
